staff/carme-casserras/cv/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>CV CARMECASS</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href="css/style.css" rel="stylesheet">
```

Comment:
@carmecass There should we no link to a css file as this project is pure html. Please remove.

Reply:
Done!

Reply:
Yep! You needed to deleted only de `link` element, not all `head` element 😄